stateoftheuniverse/widgets/sun.py

```diff
@@ -0,0 +1,39 @@
+import ephem
+
+
+# TODO:
```

Comment:
- [ ] output formatting (timezones...) - in a separate file
- [ ] return times in a different format (ISO-8601?)
- [ ] it can be done in exactly same way for other ephem objects (Moon and planets) so we could abstract this functionality to a common module
- [ ] maybe use astroplan actually, pyephem is easy to use, but not always easy to install (C library) and has a bit bizarre design